package.json

```diff
@@ -120,6 +120,9 @@
     "tmp": "^0.1.0",
     "walk-sync": "^2.0.2"
   },
+  "peerDependencies": {
+    "ember-cli": "^3.0.0"
+  },
```

Comment:
I like this so much better than white-listing the linting rule :heart:

Reply:
Yeah, totally. I used to be terrified of peerDependencies, but I've started to find more uses for them -- mostly around linting and keeping `ember-auto-import` happy 😄